### PR TITLE
Spring cleaning

### PR DESCRIPTION
**Pull request by strizr**
### Motivation

Readme updates + minor graphical update of the gallery:
- moved the categories + search above the grid, widen the grid to 3 tiles next to each other (still responsive)
- categories are now clickable (even on the tiles) - toggling filters
- added core integration flag + logo to those
- added technology property since there are redundant elements (each in different technology)

### CODE REVIEW

**Review by ChristopherJennings — APPROVED** [review]
Looks good to me!

File: README.md [hunk 1]
```diff
@@ -36,15 +78,24 @@ The [/shared](https://github.com/Kentico/kontent-custom-element-samples/tree/mas
...
+<p align="center">
+<a href="https://kentico.github.io/kontent-custom-element-samples/shared/examples.html" target="_blank"><image src="https://img.shields.io/static/v1?label=&message=show%20markdown%20examples&color=blue&style=for-the-badge" alt="visit gallery" width="250"/></a>
+</p>
+<br />
+
 
 We recommend you clone the files and host them locally yourself. The `kentico-icons-v3.0.0.woff2` file needs to be hosted in the same directory as the CSS stylesheet to be properly linked.
 
-## Feedback & Contributing
+<br />
```
Comment (Simply007) [inline README.md:91]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -82,14 +133,46 @@ The element information JSON file named after your element's name in PascalCase
...
 
 
-### Ideas
+## Ideas
 
 We'd also appreciate if you [submit your ideas](https://github.com/Kentico/kontent-custom-element-samples/issues) for custom elements or vote for [the existing ones](https://github.com/Kentico/kontent-custom-element-samples/issues).
 
 Check out the [Contributing](https://github.com/Kentico/kontent-custom-element-samples/blob/master/CONTRIBUTING.md) page to see the best places to file issues, start discussions, and begin contributing.
 
+<br />
```
Comment (Simply007) [inline README.md:147]:
```suggestion
<br />
```

File: src/pages/index.js [hunk 3]
```diff
@@ -159,7 +167,9 @@ function satisfiesSearchTextFilter(element, searchText) {
     const descriptionHasMatch = textHasMatch(element.description, searchText)
     const categories = element.categories.join(" ")
     const categoriesHasMatch = textHasMatch(categories, searchText)
-    return titleHasMatch || descriptionHasMatch || categoriesHasMatch
+    const techHasMatch = element.tech ? textHasMatch(element.tech, searchText) : false
+    const coreHasMatch = element.core ? textHasMatch("core integration", searchText) : false
```
Comment (Simply007) [inline src/pages/index.js:171]: hidden way to filter out core integrations:

**Review by Simply007 — APPROVED** [review]
Overall is good, nothing I would insist on.

I am not a fan of HTML right in the MD files, but the centered images are impossible to make in a more elegant way (but remove brake lines would be nice).

One big additional value would be to add `og:image`. You might use a social preview image from the repos and adjust it, or ask graphics, or just screen the gallery in the proper ratio and add some overlay text.

Because on FB it doesn't look good:
https://developers.facebook.com/tools/debug/?q=https%3A%2F%2Fdeploy-preview-136--kontent-custom-element-samples.netlify.app

Twitter is fine: https://cards-dev.twitter.com/validator

Linkedin is not good as well: https://www.linkedin.com/post-inspector/inspect/https:%2F%2Fdeploy-preview-136--kontent-custom-element-samples.netlify.app%2F

SEO title - I would remove "Home |" from the title

**Pull request merged**